CHANGELOG.next.asciidoc

```diff
@@ -330,6 +330,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 *Affecting all Beats*
 
+- Fingerprint processor adds a new xxhash hashing algorithm {pull}15418[15418]
```

Comment:
Would you mind moving this to the bottom of this list (so around line 389)? Generally we add new additions to the bottom.

Reply:
Ok I will do it